[2023-02-07 19:13:58,484] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: 07.1_external_dag.external_task_1 scheduled__2022-11-29T00:00:00+00:00 [queued]>
[2023-02-07 19:13:58,497] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: 07.1_external_dag.external_task_1 scheduled__2022-11-29T00:00:00+00:00 [queued]>
[2023-02-07 19:13:58,497] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-02-07 19:13:58,498] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-02-07 19:13:58,498] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-02-07 19:13:58,527] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): external_task_1> on 2022-11-29 00:00:00+00:00
[2023-02-07 19:13:58,531] {standard_task_runner.py:52} INFO - Started process 39026 to run task
[2023-02-07 19:13:58,538] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', '07.1_external_dag', 'external_task_1', 'scheduled__2022-11-29T00:00:00+00:00', '--job-id', '508', '--raw', '--subdir', 'DAGS_FOLDER/07.1_external_task.py', '--cfg-path', '/tmp/tmp0gzrcxka', '--error-file', '/tmp/tmp6n_nhfoy']
[2023-02-07 19:13:58,539] {standard_task_runner.py:80} INFO - Job 508: Subtask external_task_1
[2023-02-07 19:13:58,630] {task_command.py:371} INFO - Running <TaskInstance: 07.1_external_dag.external_task_1 scheduled__2022-11-29T00:00:00+00:00 [running]> on host pc.
[2023-02-07 19:13:58,717] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=07.1_external_dag
AIRFLOW_CTX_TASK_ID=external_task_1
AIRFLOW_CTX_EXECUTION_DATE=2022-11-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-29T00:00:00+00:00
[2023-02-07 19:13:58,718] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-02-07 19:13:58,719] {subprocess.py:74} INFO - Running command: ['bash', '-c', "sleep 10 && echo 'external task executed!'"]
[2023-02-07 19:13:58,728] {subprocess.py:85} INFO - Output:
[2023-02-07 19:14:08,730] {subprocess.py:92} INFO - external task executed!
[2023-02-07 19:14:08,731] {subprocess.py:96} INFO - Command exited with return code 0
[2023-02-07 19:14:08,766] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=07.1_external_dag, task_id=external_task_1, execution_date=20221129T000000, start_date=20230208T001358, end_date=20230208T001408
[2023-02-07 19:14:08,796] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-07 19:14:08,816] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
